synapse/http/servlet.py

```diff
@@ -331,7 +331,7 @@ def parse_string(
     default: Optional[str] = None,
     required: bool = False,
     allowed_values: Optional[Iterable[str]] = None,
-    encoding: str = "ascii",
+    encoding: str = "UTF-8",
```

Comment:
I don't think we want to do this for all callers of `parse_string`. I think we want to change the call in the admin API to include `encoding="utf-8"`.

Reply:
I suspect there's other calls to this that *really* do actually want only ASCII strings. 😄

Reply:
Yes that makes way more sense than what I did.